README.md

```diff
@@ -1,35 +1,20 @@
-# OSM_DataManipulation
+#Sateltiles
```

Comment:
Prefer space for comment like `# this is comment` and not like `#this is comment`